### ISSUE TITLE
[Enhancement] an enhancement about the error message of mindspore.ops.permute

### ISSUE THREAD

**Issue by Redmept1on** [mindspore-ai/mindspore#268]
## Background
The error message should be more explicit when the axis parameter of mindspore.ops.permute has the same dimension.

## Introduction
Maybe follow pytorch

mindspore
```python
import mindspore as mind
import numpy as np

a = mind.tensor(np.array([[1, 2, 3], [float('nan'), 5, 6]]),dtype=mind.float32)
max1 = mind.ops.permute(a,(0, 0))
print(max1)
```
![image](https://github.com/mindspore-ai/mindspore/assets/84373897/80141263-eb60-41a6-8d0b-15c0bd13cf81)

pytorch
```python
import torch
import numpy as np

a = torch.tensor(np.array([[1, 2, 3], [float('nan'), 5, 6]]),dtype=torch.float32)
max1 = torch.permute(a,(0, 0))
print(max1)
```
![image](https://github.com/mindspore-ai/mindspore/assets/84373897/7ffb2b32-6656-4868-93b0-fefc6051a41c)

**Comment by 1071034024** [comment]
Thanks for your suggestion, we will fix it in the next version
